### PR TITLE
#260 update client for new scylla

### LINKED ISSUE #260: Update Client to Work with new DataTypeName format
### Description

Sending the dataTypeName in the new format with the nodeName broke some things in the frontend. Additionally, not sending the list of node names to the frontend broke the graph page. This ticket is to get all that stuff working again, 

### Acceptance Criteria

- [ ] Frontend has the full functionality it had before the recent Scylla changes

### Proposed Solution

Code

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

Update client to maintain all old functionality with the new changes to Scylla:
- Transforms list of all datatypes on the graph page to fit the node structure we were using before (I like this a lot better because any weirdness related to nodes/ how we display datatypes is localized to the graph page, so you only have to change it in one place now to change how its's displayed. This way, when/if y'all do the multiple levels of tabs on the graph page, it should be significantly easier/ only involve frontend changes on the graph page)
- Update the DataType enum to include nodes (aka, the dataTypes are basically just topics now)

Clean up the DataType enum file, rename it, and move its file to the src/ folder, so it's not hidden away since we use it in basically all of our custom components that aren't the graph page

Also includes an update to Scylla so that the dependencies play nice on Windows (removes ability to connect to the database over TLS from Scylla, but should be no biggy)

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `package-lock.json` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #260  (issue #)

### CODE REVIEW

**Comment by RChandler234** [comment]
Make sure to update the READMEs before the semester starts so that all the dev setup instructions are gtg and work on all platforms @bracyw, for Scylla and the Client, had some small issues with Scylla when testing this

**Review by jr1221 — COMMENTED** [review]
tested.  100% works.

File: angular-client/src/data-type.enum.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+export enum DataTypeEnum {
+  DRIVER = 'Driver',
+  LOCATION = 'location',
+  POINTS = 'GPS/Location',
```
Comment (jr1221) [inline angular-client/src/data-type.enum.ts]: TPU/GPS/Location

File: angular-client/src/data-type.enum.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  NEW_LATENCY = 'Latency',
+
+  //Fake Data Points
+  MOTOR_USAGE = 'Motor Usage',
+  COOL_USAGE = 'Cooling Usage',
+  STEERING_ANGLE = 'Steering Angle',
+  TORQUE = 'Torque',
+  BRAKE_PRESSURE = 'Brake Pressure',
+  ACCELERATION = 'Acceleration',
+  XYZAccel = 'XYZAcceleration',
```
Comment (jr1221) [inline angular-client/src/data-type.enum.ts:17]: @bracyw technically some of this info is being made available, but it's not a simple single datapoint, wanna figure this out rq in this PR or wait?

Comment (jr1221) [reply angular-client/src/data-type.enum.ts:17]: Eh let's wait it's pretty wierd. Basically there are 4 accelerations.

**Review by bracyw — DISMISSED** [review]
lgtm

**Pull request merged**